Mathlib/Analysis/CStarAlgebra/CStarMatrix.lean

```diff
@@ -579,6 +579,9 @@ instance instT3Space : T3Space (CStarMatrix m n A) := _root_.instT3Space
 instance instIsTopologicalAddGroup : IsTopologicalAddGroup (CStarMatrix m n A) :=
   Pi.topologicalAddGroup
 
+@[deprecated (since := "2025-14-02")] alias instTopologicalAddGroup :=
+  instIsTopologicalAddGroup
```

Comment:
If I understand correctly, we don't deprecate instance names - right? (So, you can remove all those aliasses again.)

Reply:
I'll trust you on this, we can always revert the commit if it turns out you're wrong :-)